### PR TITLE
Filtering of Applicants in HRM

### PR DESCRIPTION
**Pull request by dalex135**
Please test this really well. and let me know if there is any bug.

### CODE REVIEW

File: Backend/src/main/java/Digitact/Backend/ConfigProperties.java [hunk 1]
```diff
@@ -7,7 +7,7 @@
 @Configuration
 public class ConfigProperties {
     public static final int serverPort = 9090;
-    public static final String absoluteFileSystemPath = "";
+    public static final String absoluteFileSystemPath = "D:\\Masters\\2nd Semester\\AMOS Project\\FileSystem\\";
```
Comment (vishwasanavatti) [inline Backend/src/main/java/Digitact/Backend/ConfigProperties.java]: Make sure you dont push this change.

File: HR-Monitor/src/Main/Configuration.java [hunk 2]
```diff
@@ -6,7 +6,7 @@
 public class Configuration {
     public static final String BES_URI = "http://localhost:9090/api/HRController/";
 
-    public static final String absoluteFileSystemPath = "";
+    public static final String absoluteFileSystemPath = "D:\\\\Masters\\\\2nd Semester\\\\AMOS Project\\\\FileSystem\\\\";
```
Comment (vishwasanavatti) [inline HR-Monitor/src/Main/Configuration.java]: same here! dont merge this change.

File: HR-Monitor/src/Util/Dictionary/IDictionary.java [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+        if (LANG.equals(Language.English)) {
+            if (english.contains("TITLE-")) // GUI title
+            return english.replace("TITLE-", "");
+            return english;
+        }
+
+        return getGerman(dictionary, english);
+    }
+
+    private static String getGerman(IDictionary dictionary, String english) {
```
Comment (vishwasanavatti) [inline HR-Monitor/src/Util/Dictionary/IDictionary.java]: shouldn't the access modifier be public?

**Review by vishwasanavatti — COMMENTED** [review]
Great job, the filter thing just works perfect. Also the translation between German and English works great as well. I found alignment issues at 2 places due to translation and also have a question. 
Filter works fine but is it good idea to call a query at each select or entry in the field? it just keeps calling query and affects the performance I fell.  I prefer to have a search button and on click of it a query will be called only once and data will be fetched.

File: Backend/src/main/java/Digitact/Backend/ConfigProperties.java (hunk 1 above)
Comment (dalex135) [reply Backend/src/main/java/Digitact/Backend/ConfigProperties.java]: Yep. these will be corrected in the last commit when I am about to merge.

**Comment by dalex135** [comment]
@vishwasanavatti I corrected the DB calls for each and the German translations. 
@denoribi Pls check for the German strings here. If check here that's enough. This PR has all the changes regarding to translations.

**Review by denoribi — APPROVED** [review]
Translations look correct to me :)
Good job guys ;)

**Comment by Nasser-Eddin-Nasser** [comment]
I ran the code locally and checked each commit. It works perfectly, just a small issue, by german version the position and industry still in the English version. 
image

**Review by vishwasanavatti — APPROVED** [review]
Checked locally and it works fine. ok to merge.

**Pull request merged**